content/en/docs/tasks/administer-cluster/memory-manager.md

```diff
@@ -211,6 +213,8 @@ The following means can be used to troubleshoot the reason why a pod could not b
 - pod status - indicates topology affinity errors
 - system logs - include valuable information for debugging, e.g., about generated hints
 - state file - the dump of internal state of the Memory Manager (includes [Node Map and Memory Maps][2]) 
+- starting from v1.22, the Resource API can be used to retrieve information about the memory reserved for containers

```

Comment:
What resource API? Can I `kubectl get Resource` (readers might think the answer is Yes). Can we clarify - link to the section below to help explain.

Reply:
Yes, adding a link would be helpful.

Reply:
Not sure if accessible via `kubectl` (most likely not). @cynepco3hahue Could you confirm?

Some snippet code from @cynepco3hahue to utilize the API:

```
package main
import (
	"context"
	"encoding/json"
	"fmt"
	"os"
	"time"
	kubeletpodresourcesv1 "k8s.io/kubelet/pkg/apis/podresources/v1"
	"k8s.io/kubernetes/pkg/kubelet/apis/podresources"
)
const (
	defaultSocketPath          = "unix:///var/lib/kubelet/pod-resources/kubelet.sock"
	defaultPodResourcesTimeout = 10 * time.Second
	defaultPodResourcesMaxSize = 1024 * 1024 * 16 // 16 Mb
)
func main() {
	cli, conn, err := podresources.GetV1Client(defaultSocketPath, defaultPodResourcesTimeout, defaultPodResourcesMaxSize)
	if err != nil {
		panic(err)
	}
	defer conn.Close()
	resp, err := cli.List(context.TODO(), &kubeletpodresourcesv1.ListPodResourcesRequest{})
	if err != nil {
		panic(err)
	}
	json.NewEncoder(os.Stdout).Encode(resp)
	allocResp, err := cli.GetAllocatableResources(context.TODO(), &kubeletpodresourcesv1.AllocatableResourcesRequest{})
	if err != nil {
		panic(err)
	}
	for _, device := range allocResp.Devices {
		for _, node := range device.Topology.Nodes {
			fmt.Printf("NUMA node ID: %d\n", node.GetID())
		}
	}
	json.NewEncoder(os.Stdout).Encode(allocResp)
}
```

Reply:
In general it is the pod resource API, and you can create a section similar to https://kubernetes.io/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins/#monitoring-device-plugin-resources or just link to it